themes/cardinal_service/lib/components/minimal-card/minimal_card.scss

```diff
@@ -0,0 +1,9 @@
+@charset 'UTF-8';
+
+@import '../../scss/config';
+
+.su-minimal-card {
```

Comment:
The link should be centered vertically depending on the size of the image. I believe that can be accomplished with flexbox but I can't remember.